hypervisor/include/public/acrn_common.h

```diff
@@ -50,7 +50,8 @@
 #define SECURE_WORLD_ENABLED    (1UL << 0U)	/* Whether secure world is enabled */
 #define LAPIC_PASSTHROUGH	(1UL << 1U)  	/* Whether LAPIC is passed through */
 #define IO_COMPLETION_POLLING	(1UL << 2U)  	/* Whether need hypervisor poll IO completion */
-#define CLOS_REQUIRED          (1UL << 3U)     /* Whether CLOS is required */
+#define CLOS_REQUIRED		(1UL << 3U)     /* Whether CLOS is required */
+#define HIDE_MTRR		(1UL << 4U)  	/* Whether hide MTRR from VM */
```

Comment:
Can we format those bits to have a same prefix? GUEST_FLAG_HIDE_MTRR ? And also update other bits.

Reply:
Sure